src/internet_identity/src/storage/tests.rs

```diff
@@ -0,0 +1,175 @@
+use crate::storage::{Header, StorageError};
```

Comment:
(about the whole module) if I understand correctly, these tests make use of a dummy vector-based implementation for stable memory, correct? Also, those tests are compiled and run as x86/aarch, right? Would there be any reason why memory/serializing would behave different when compiled to Wasm?

Reply:
Yes, exactly. The only potential difference I see is that for Wasm we have a 32bit architecture and on x86/aarch we have 64bit. This means that the size of `usize` is 8 instead of 4 in the tests. But since the `Memory` trait uses `u64` exclusively, this should not be an issue.